scripts/fastlane/Fastfile

```diff
@@ -26,6 +28,7 @@ platform :android do |options|
   lane :create_next_release_pr do |options|
     # This will be used in the commit and PR title
     product_name = options[:product_name]
+    maven_group_name = options[:maven_group_name]
```

Comment:
Looks like the package name is defined [in the Appfile, here](https://github.com/aws-amplify/amplify-android/blob/main/scripts/fastlane/Appfile#L2), so another option might be to grab the value from there using: 

```
maven_group_name = CredentialsManager::AppfileConfig.try_fetch_value(:package_name)
```

Ref: https://docs.fastlane.tools/advanced/Appfile/

I'm not sure if that's better or not, but it's another option.

Reply:
Let me try it. It would be one less thing to pass via the command line.

Reply:
That worked! Just updated the PR